sidh/sidhp751_interface.go

```diff
@@ -0,0 +1,75 @@
+// sidhp751_interface implements all the required interface APIs for performing
+// SIDH key-exchange over p751 parameter. It imports the sidhp751 implementation
+// and converts all the underlying data structures to generic interface structures.
+// Author: Amir Jalali		ajalali2016@fau.edu
+package sidh
```

Comment:
This code should probably be in the sidhp751 package.

Reply:
Actually, the functions GenerateAliceKeypair and such are generic so should be in this package, but the `func (e *p751SIDH)` functions are generic and don't need to be written because they can be defined generically on the SIDH struct in sidh.go.